crates/p2p_proto/src/lib.rs

```diff
@@ -31,6 +31,12 @@ impl ToProtobuf<u32> for u32 {
     }
 }
 
+impl ToProtobuf<u8> for u8 {
```

Comment:
Does `protobuf` really need a hint to know that `u8` is `u8`? Weird...

Reply:
This is due to `impl for Vec<T>`